docs/content/api_core/api_installing.rst

```diff
@@ -6,7 +6,7 @@ Getting Started with SimPEG
 Dependencies
 ============
 
-- Python 2.7
+- Python 3.5 (or 2.7)
```

Comment:
Minor updates. More can come in through the tutorials materials, however, some of those are now out of date.